packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -530,6 +540,8 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
 
   @override
   void initState() {
+    debugPrint('gesture_detector ' + widget.dragStartBehavior.toString());
+    debugPrint('name ' + this.toString());
```

Comment:
these presumably need to go before checkin

Reply:
yup thanks!